src/main/java/org/traccar/model/Command.java

```diff
@@ -45,6 +45,7 @@ public class Command extends BaseCommand {
     public static final String TYPE_SILENCE_TIME = "silenceTime";
     public static final String TYPE_SET_PHONEBOOK = "setPhonebook";
     public static final String TYPE_MESSAGE = "message";
+    public static final String TYPE_ROLLCALL = "rollcall";
```

Comment:
What does this command do?

Reply:
[Function] look over the current position of the designated vehicle.
- From the protocol documentation.

Reply:
Why are we not using `TYPE_POSITION_SINGLE` for it?

Reply:
We can use TYPE_POSITION_SINGLE too. Rollcall was specific to gator protocol but we can use more generic name.

I will update the codebase.

Reply:
@tananaev Changed  ROLLBACK to `TYPE_POSITION_SINGLE`